appbars/src/main/res/values/styles-topbars.xml

```diff
@@ -52,4 +52,25 @@
         <item name="srcCompat">@drawable/ic_vtmn_more_2_line</item>`
         <item name="android:src">@drawable/ic_vtmn_more_2_line</item>
     </style>
+
+    <!-- Customization of the search view inside a Toolbar -->
+    <eat-comment />
+
+    <style name="Widget.Vitamin.TopBar.Search" parent="@style/Widget.Vitamin.TopBar">
+        <item name="contentInsetStart">0dp</item>
+        <item name="materialThemeOverlay">@style/ThemeOverlay.Vitamin.TopBar.Search</item>
+    </style>
+
+    <style name="ThemeOverlay.Vitamin.TopBar.Search" parent="">
+        <item name="colorOnPrimary">?attr/vtmnContentTertiary</item>
+    </style>
+
+    <style name="Widget.Vitamin.SearchView" parent="@style/Widget.AppCompat.SearchView">
+        <item name="closeIcon">@drawable/ic_vtmn_close_line</item>
```

Comment:
The icons (the magnifying glass and the cross) appear grey. It's not black as the topBar. Are we ok with that ?
![Capture d’écran 2022-01-20 à 15 35 10](https://user-images.githubusercontent.com/46402524/150359128-5c5c2f25-e312-4048-93f5-23c863d81fc0.png)

Reply:
The search icon is grey in the specification. Unfortunately, I can't change the color of the close button to black...